lc0@exe/lc0@exe.log

```diff
@@ -0,0 +1,2 @@
+﻿  layers.cc
+  lc0@exe.vcxproj -> C:\Ankan\git\ankan\lc0\build\.\lc0.exe
```

Comment:
What is this file? :)

Reply:
Sorry. Likely some intermediate build file that accidentally got submitted. Will remove it.